pandas/tests/frame/test_operators.py

```diff
@@ -972,7 +972,7 @@ def test_boolean_comparison(self):
         assert_numpy_array_equal(result, expected.values)
 
         self.assertRaises(ValueError, lambda: df == b_c)
-        self.assertFalse((df.values == b_c))
+        self.assertFalse(np.array_equal(df.values, b_c))
```

Comment:
Can `assert_numpy_array_equal` check if two arrays are not equal?  The statement is asserting that they are not.